src/Compilers/CSharp/Portable/Binder/Binder_Deconstruct.cs

```diff
@@ -608,6 +608,12 @@ private static void FlattenDeconstructVariables(ArrayBuilder<DeconstructionVaria
 
         internal BoundLocalDeconstructionDeclaration BindDeconstructionDeclarationStatement(DeconstructionDeclarationStatementSyntax node, DiagnosticBag diagnostics)
         {
+            var constKeyword = node.Modifiers.FirstOrDefault(SyntaxKind.ConstKeyword);
+            if (constKeyword != default(SyntaxToken))
```

Comment:
@cston I'll use the variable component for now, as I agree we should be consistent.
@gafter is there a problem with putting the squiggle on the wild cards "(*, *)"?